### PR TITLE
Fix sheduled issue in case of reset

### PR DESCRIPTION
**Pull request by vdoleans01**

### CODE REVIEW

File: session-replacement/src/main/java/com/amadeus/session/ExecutorFacade.java [hunk 1]
```diff
@@ -232,7 +233,9 @@ public void uncaughtException(Thread t, Throwable e) {
   public void shutdown() {
     logger.info("Shutting down the executor.");
     executor.shutdown();
+    executor.shutdownNow();
     scheduledExecutor.shutdown();
+    scheduledExecutor.shutdownNow();
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/ExecutorFacade.java]: Do we need shutdown before shutdownNow? I think the later should be enough.

File: session-replacement/src/main/java/com/amadeus/session/servlet/SessionFilter.java [hunk 2]
```diff
@@ -81,21 +82,45 @@ public void initForSession(FilterConfig config) {
...
+
+      logger.debug("Start of SessionFilter doFilter ");
       // Call next filter in chain
       chain.doFilter(request, response);
+
+      logger.debug("after the doFilter of SessionFilter doFilter ");
+
+    } catch (Exception e) {
+
+      logger.error("Error in sessionFiler catch", e);
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/servlet/SessionFilter.java]: typo "Session filter" + maybe we should not log stack trace here as the exception is propagated.

Comment (vdoleans01) [reply session-replacement/src/main/java/com/amadeus/session/servlet/SessionFilter.java]: typo fixed 
i prefer to keep the logging

**Comment by nbogojevic** [comment]
LGTM

**Pull request merged**